pkg/virt-api/webhooks/validating-webhook/admitters/migration-create-admitter_test.go

```diff
@@ -289,3 +230,21 @@ var _ = Describe("Validating MigrationCreate Admitter", func() {
 		)
 	})
 })
+
+func newAdmissionReviewForVMIMCreate(migration *v1.VirtualMachineInstanceMigration) (*admissionv1.AdmissionReview, error) {
```

Comment:
Please consider introducing this change in a dedicated commit.

Reply:
Done.